js/tooltip.js

```diff
@@ -385,13 +385,36 @@
     return delta
   }
 
+  Tooltip.prototype.didntReceiveTypeStringOrFunction = function (received) {
+    return !/string|function/.test(received)
+  }
+
+  Tooltip.prototype.didntReturnStringOrJQuery = function (returned, returnedType) {
+    return (returnedType !== 'string' && !(returned.jquery))
+  }
+
```

Comment:
Personally, I would move the `throw`s into these functions and rename them along the lines of `checkValidContentType`